components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/SQLQueries.java

```diff
@@ -957,13 +957,13 @@
             "SELECT SCOPES.SCOPE_ID, SCOPES.NAME, SCOPES.DISPLAY_NAME, SCOPES.DESCRIPTION, " +
                     "SCOPEBINDINGS.SCOPE_BINDING, SCOPEBINDINGS.BINDING_TYPE FROM IDN_OAUTH2_SCOPE SCOPES " +
                     "LEFT JOIN IDN_OAUTH2_SCOPE_BINDING SCOPEBINDINGS ON SCOPES.SCOPE_ID=SCOPEBINDINGS.SCOPE_ID " +
-                    "WHERE SCOPES.TENANT_ID=? AND SCOPES.SCOPE_TYPE=? AND SCOPES.NAME IN (?)";
+                    "WHERE SCOPES.TENANT_ID=? AND SCOPES.SCOPE_TYPE=? AND SCOPES.NAME IN (_SCOPE_LIST_)";
```

Comment:
let's use a constant for `_SCOPE_LIST_` and reuse it in other places

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1603/commits/9a22c8f2df57eb1ad51c7db8e3b7e63b971a03e1